sympy/series/sequences.py

```diff
@@ -716,14 +714,13 @@ def length(self):
 
 
 class SeqAdd(SeqExprOp):
-    """
-    Represents term-wise addition of sequences
+    """Represents term-wise addition of sequences.
 
     Rules:
         * The interval on which sequence is defined is the intersection
-        of respective intervals of sequences.
-        * Anything + EmptySequence, remains unchanged
-        * Other rules are defined in _add methods of sequence classes
+            of respective intervals of sequences.
```

Comment:
It's visible as following: (extra indented)

![leo](https://cloud.githubusercontent.com/assets/5647941/8593631/1fdd8352-2657-11e5-991e-1cd171b85f8c.png)